plugins/utils/project-nvim.nix

```diff
@@ -90,9 +90,15 @@ in
   };
 
   config = mkIf cfg.enable {
-    warnings = optional (cfg.enableTelescope && (!config.plugins.telescope.enable)) ''
```

Comment:
Is directly updating from warning to assertion ok, or should there be a transition period?

Reply:
I feel like this could easily be upgraded, now. We added the warning about 5 months ago so its had a good warning period.